### PR TITLE
mongodb 3.0.1

### PR DESCRIPTION
**Pull request by ehershey**

### CODE REVIEW

File: Library/Formula/mongodb.rb [hunk 1]
```diff
@@ -16,10 +16,6 @@ class Mongodb < Formula
   end
 
   bottle do
-    revision 1
-    sha256 "e9bb9dce2bc6e728d93e3c8891dba77b665c4928918fc5de6e8ee853431d79d5" => :yosemite
-    sha256 "d292c96a2ea24b3be324b89412cd0a38403afddb00c6b9d17b550f58070d1207" => :mavericks
-    sha256 "80ccc1c2e3b3e334bafdbd0efdb55f783fd8a4f5f4f440dd08daa1db9332601e" => :mountain_lion
```
Comment (tdsmith) [inline Library/Formula/mongodb.rb]: Just FYI, you can leave the bottle stanza alone when preparing a PR.

**Comment by tdsmith** [comment]
`mongo-tools` has a new release; should those be updated as well? https://github.com/mongodb/mongo-tools/releases

File: Library/Formula/mongodb.rb (hunk 1 above)
Comment (ehershey) [inline Library/Formula/mongodb.rb]: Oh, cool, thanks.

**Comment by bmike** [comment]
Is there something I can do to help get this merged?

**Comment by mikemcquaid** [comment]
@ehershey @bmike I guess if someone addressed the mongo-tools question either in this PR or a new one?

**Comment by bmike** [comment]
I don't really get what mongo-tools is or does. It's not part of the mongodb tarball v3.0.1 and has a separate delivery and versioning track AFAIK.

* https://jira.mongodb.org/browse/TOOLS/?selectedTab=com.atlassian.jira.jira-projects-plugin:versions-panel

3.0.1 version of the tools are bundled in the 3.0.1 database pagkage and the tools seems to be the source code (mostly go) to recompile the 3.0.1 version of the tarball. The 3.0.0 version of mongodb on homebrew live today has none of these tools source code so I must be missing something basic here as I'm not seeing how or why the 3.0.1 database and tools binaries would be on hold.

**Pull request closed without merging**